src/MovieDisplay.css

```diff
@@ -1,26 +1,53 @@
 .movieBackdrop {
   background-size: cover;
   background-repeat: no-repeat;
-  height: 100vh;
-  width: 100vw;
+  /* opacity: 0.5; */
+  /* height: 100vh;
+  width: 100vw; */
+}
+
+.movieContainer {
+  height: 90vh;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
 }
 
 .moviePoster {
   height: 400px;
   width: 300px;
+  border-radius: 7px;
 }
 
 .movieInfo {
   height: 400px;
-  width: 300px;
-  border: 2px solid orange;
-  background-color: rgb(234, 160, 22);
+  width: 50%;
+  /* background-color: #FCAB10; */
+  background-image: linear-gradient(to bottom right, #FCAB10, white);
+  /* background-image: linear-gradient(to bottom left, #44AF69, #FCAB10); */
+  /* background-image: linear-gradient(to bottom left, #44AF69, #2B9EB3); */
+  border-radius: 7px;
+  padding: 5px 10px 5px 15px;
+  /* display: flex;
+  flex-direction: column; */
+  /* align-content: center; */
+  /* color: white; */
 }
 
-.movieContainer {
-  height: 100vh;
-  display: flex;
-  justify-content: space-evenly;
-  align-items: center;
-  border: 2px solid purple;
-}
\ No newline at end of file
+.infoTitle {
+  font-size: 17px;
+  font-family: fantasy;
```

Comment:
I think we could refactor all the family fonts in CSS using a wildcard selector ✳️